runtime/doc/api.txt

```diff
@@ -3042,6 +3042,12 @@ nvim_open_win({buffer}, {enter}, {*config})                  *nvim_open_win()*
                       specified by character: [ {"+", "MyCorner"}, {"x",
                       "MyBorder"} ].
 
+                  • title: Title (optional) in window border, keys
+                    • "title_text": text in borer.
```

Comment:
Perhaps nested keys should skip `title_` prefix?

Reply:
not sure for this. maybe we don't need but check `buf_set_extmark` it use `virt_text virt_text_pos `

Reply:
That because `virt_text` and `virt_text_pos` are member of the same dictionary, while here `title_text` is a member of `title`.

Reply:
oh so remove the prefix is recommend right?